src/app/clusters/valve-configuration-and-control-server/valve-configuration-and-control-server.cpp

```diff
@@ -145,7 +111,8 @@ ValveConfigAndControlAttrAccess gAttrAccess;
 CHIP_ERROR ValveConfigAndControlAttrAccess::ReadRemainingDuration(EndpointId endpoint, AttributeValueEncoder & aEncoder)
 {
     DataModel::Nullable<uint32_t> rDuration;
-    VerifyOrReturnError(GetRemainingDuration(endpoint, rDuration), CHIP_IM_GLOBAL_STATUS(UnsupportedAttribute));
+    VerifyOrReturnError(ValveConfigurationAndControl::GetRemainingDuration(endpoint, rDuration) == CHIP_NO_ERROR,
```

Comment:
Why does every error result in UnsupportedAttribute? Should it not happen only for something like `NOT_IMPLEMENTED` or similar?